src/foam/nanos/auth/Country.js

```diff
@@ -28,7 +28,9 @@ foam.CLASS({
     name: 'toString',
     returns: 'String',
     javaReturns: 'String',
-    code: function() { return "Country: " + this.code + ", " + this.name; },
+    code: function() {
+      return 'Country: ' + this.code + ', ' + this.name; 
```

Comment:
User string interpolation here.

Reply:
Interpolation used to be about 100 times slower, but I just ran a benchmark and it seems that interpolation is as fast or faster now.

Reply:
But I wouldn't bother changing existing code to use interpolation.